### DISCUSSION TITLE
[HEO主题] Safari浏览器波浪动效性能

### DISCUSSION THREAD

**Discussion by priforjm** [tangly1024/NotionNext#1968] [Q&A]
**复现步骤**
文章页面上方这个波浪特效在safari浏览器里非常卡，在chrome, edge都正常

**期望的正常结果**

**截图**
<img width="560" alt="Screenshot 2023-07-26 at 12 48 02" src="https://github.com/tangly1024/NotionNext/assets/131368755/40188aac-c02c-4856-a874-a2a1485aece3">

**环境**

- 操作系统:  macOS
- 浏览器 safari
- NotionNext版本 [e.g. 3.13.6]
- 主题 HEO

**补充说明**
与问题相关的其它说明

**Comment by Angel0726** [comment]
手机会有一些卡顿。但是电脑会好一点
